src/components/common/BarChart.tsx

```diff
@@ -2,10 +2,22 @@ import * as React from 'react';
 import { makeStyles, Theme } from '@material-ui/core/styles';
 import { smallFontSize } from 'components/Theme/constants';
 import { COLOR_SPECTRUM } from 'components/Theme/colorSpectrum';
-import { Tooltip, Zoom } from '@material-ui/core';
+import { Tooltip, Typography, Zoom } from '@material-ui/core';
 
 const useStyles = makeStyles((theme: Theme) => ({
-  container: {
+  chartContainer: {
+    paddingLeft: theme.spacing(1),
+    paddingRight: theme.spacing(3),
+    paddingTop: theme.spacing(1),
+    minHeight: '135px',
```

Comment:
added min height, so the chart is rendered inside the reserved space without any shifts of the content below it